.buildkite/premerge.steps.yaml

```diff
@@ -132,3 +139,66 @@ steps:
       WORKER_TYPE: "UnityGameLogic"
       BUILD_ENVIRONMENT: "cloud"
       SCRIPTING_BACKEND: "mono"
+
+# Need to duplicate steps to be able to run it on Windows & macOS...
+# https://forum.buildkite.community/t/steps-with-a-matrix/46/2
+
+  - label: ":darwin: ~ test"
+    command: bash -c ci/test.sh
+    <<: *macos
+    artifact_paths:
+      - logs/**/*
+
+  - label: ":darwin: ~ build :android:"
+    command: bash -c .shared-ci/scripts/build-worker.sh
+    <<: *macos
+    artifact_paths:
+      - logs/**/*
+      - workers/unity/build/worker/**/*
+    env:
+      WORKER_TYPE: "MobileClient"
+      BUILD_ENVIRONMENT: "local"
+      BUILD_TARGET_FILTER: "android"
+      SCRIPTING_BACKEND: "mono"
+
+  - label: ":darwin: ~ build :ios:"
+    command: bash -c .shared-ci/scripts/build-worker.sh
+    <<: *macos
+    artifact_paths:
+      - logs/**/*
+    env:
+      WORKER_TYPE: "MobileClient"
+      BUILD_ENVIRONMENT: "local"
+      BUILD_TARGET_FILTER: "ios"
+      SCRIPTING_BACKEND: "il2cpp"
+      TARGET_IOS_SDK: "simulator"
```

Comment:
Do we want to build for device too? Or does that require some extra setup

Reply:
not yet. as soon as we can sign applications in buildkite: yes